package.json

```
@@ -70,6 +71,26 @@
         "command": "runme.resetEnv",
         "category": "Runme",
         "title": "Reset Environment Variables for current VS Code session"
+      },
+      {
+        "command": "runme.resetEnv",
+        "category": "Runme",
+        "title": "Reset Environment Variables for current VS Code session"
+      },
+      {
+        "command": "runme.openAsRunmeNotebook",
+        "category": "Runme",
+        "title": "Open markdown as runme",
```

Also should we avoid showing this command in the command palette? I assume this command only makes sense in certain context so we should ensure it is only shown there.

```suggestion
        "title": "Open markdown as Runme",
```